modules/videosipgw/VideoSIPGW.js

```diff
@@ -0,0 +1,144 @@
+import { getLogger } from 'jitsi-meet-logger';
+const logger = getLogger(__filename);
+
+import JitsiVideoSIPGWSession from './JitsiVideoSIPGWSession';
+import * as Constants from './VideoSIPGWConstants';
+import XMPPEvents from '../../service/xmpp/XMPPEvents';
+
+/**
+ * Main video SIP GW handler. Stores references of all created sessions.
+ */
+export default class VideoSIPGW {
+
+    /**
+     * Creates new handler.
+     *
+     * @param {ChatRoom} chatRoom - Tha chat room to handle.
+     */
+    constructor(chatRoom) {
+        this.chatRoom = chatRoom;
+        this.eventEmitter = chatRoom.eventEmitter;
+        logger.info('creating VideoSIPGW');
+        this.sessions = {};
+
+        this.sessionStateChangeListener = this.sessionStateChanged.bind(this);
+
+        chatRoom.addPresenceListener('jibri-sip-status',
+            this.handleJibriSIPStatus.bind(this));
+        chatRoom.addPresenceListener('jibri-sip-call-state',
+            this.handleJibriSIPState.bind(this));
+    }
+
+    /**
+     * Handles presence nodes with name: jibri-sip-status.
+     *
+     * @param node the presence node to handle.
```

Comment:
Missing "node" type. If it's hard to tell then maybe mark as {Object} and explain whether it's a DOM Element or what is it (I don't know) ?